pdfminer/layout.py

```diff
@@ -522,13 +563,97 @@ def __init__(self, bbox):
         self.groups = None
         return
 
-    # group_objects: group text object to textlines.
-    def group_objects(self, laparams, objs):
+    def hsplitted(self, obj0, obj1, splitobjs, cell_margin):
+        ''' detect that between obj0 and obj1 there is a horizontal
+        split line.
+
+        # ARGUMENTS:
+            obj0: a LTTextLineHorizontal, LTTextLineVertical, or LTChar Object
+            obj1: a LTTextLineHorizontal, LTTextLineVertical, or LTChar Object
+            splitobjs: list -- each elememnt is a LTRect Object
+            cell_margin: float -- the distance given for allowing
+                additional matches
+
+        # RETURNS:
+            boolean
+        '''
```

Comment:
Props for added documentation! But use double quotes for docstrings and `:param obj0:` style for documenting arguments and return values.

Reply:
Thank you!  I have moved over documentation to now match the :param ..: style.